nixos/modules/services/mail/mailhog.nix

```diff
@@ -75,5 +80,13 @@ in
         StateDirectory = "mailhog";
       };
     };
+
+    services.mail.sendmailSetuidWrapper = lib.mkIf cfg.setSendmail {
+      program = "sendmail";
+      source = "${mhsendmail}/bin/mailhog-sendmail";
+      # Communication happens through the network
+      owner = "nobody";
```

Comment:
If mailhog wouldn't use dynamicuser, we would have a user we can use as nobody shouldn't really be used

Reply:
I am not sure if we should do this

Reply:
@peterhoeg fyi

Reply:
When application use local storage to save a bigger amount of data, DynamicUser shouldn't be used anyway.

Reply:
@SuperSandro2000 should I also create a PR to revert the DynamicUser change? Because I believe this is slighlty unrelated to this PR

Reply:
This wrapper does not write to FS. It just connecting to the service, sending the mail and exiting. Is there a better user/groups to use or is nobody fine?